src/ray/object_manager/pull_manager.cc

```diff
@@ -326,6 +326,11 @@ void PullManager::TryToMakeObjectLocal(const ObjectID &object_id) {
   if (request.next_pull_time > get_time_()) {
     return;
   }
+  if (request.is_being_restored) {
```

Comment:
We still have 10ms interval retry in this case? And the only difference is we don't send a RPC right?

Reply:
Yes, that is correct, this object won't be removed from the active pull request table while the restoration request is pending, but will instead short-circuit here on each retry timer tick.